src/symbol.cr

```diff
@@ -14,10 +14,19 @@
 struct Symbol
   include Comparable(Symbol)
 
+  # Compares symbol with other_symbol after calling `to_s` on each of the symbols.
+  # Returns -1, 0 or +1 depending on whether symbol is less than, equal to, or
+  # greater than other_symbol.
+  # See `String#<=>` for more information.
   def <=>(other : Symbol)
     to_s <=> other.to_s
   end
 
+  # Returns the representation of sym as a symbol literal.
```

Comment:
Could be slightly misleading since there's in fact `Macro::SymbolLiteral`. Perhaps "Returns the symbol literal representation as a string"?